source/2020-12-18-the-ember-times-issue-170.md

```diff
@@ -76,14 +77,21 @@ The blog post is a step-by-step explanation with extensive code examples of usin
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Glimmer apps with Snowpack 🔥️⛄](https://dev.to/rajasegar/building-glimmer-apps-with-snowpack-igc)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+There's an awesome new [blog post](https://dev.to/rajasegar/building-glimmer-apps-with-snowpack-igc) from [Rajasegar Chandran @rajasegar](https://github.com/rajasegar) about how to build your Glimmer app with Snowpack 🔥️⛄!
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The Glimmer VM runs the components we know and 💜 in Ember, and it is designed to run them super fast 🔥. Snowpack ⛄️ is a new, lightning-fast frontend build tool designed for the modern web. Snowpack leverages JavaScript's native module system [(ESM)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import).
```

Comment:
I think we can skip mentioning the abbreviation and add the link to "JavaScript's native module system," which may be more descriptive.

```suggestion
The Glimmer VM runs the components we know and 💜 in Ember, and it is designed to run them super fast 🔥. Snowpack ⛄️ is a new, lightning-fast frontend build tool designed for the modern web. Snowpack leverages [JavaScript's native module system](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import).
```